### PR TITLE
Added ScapyCon to Conferences

### PR DESCRIPTION
**Pull request by polybassa**

### CODE REVIEW

**Comment by polybassa** [comment]
@guedou @gpotter2 Can I merge this?

File: src/views/Home.vue [hunk 1]
```diff
@@ -529,6 +529,17 @@ const RESOURCES = [
...
 /* Conferences */
 const CONFERENCES = [
+  {
+    "title": "ScapyCon Automotive",
+    "venue": "ScapyCon 2024",
+    "links": [
+      {
+        tooltip: "Website",
+        icon: mdiWeb,
+        link: "https://scapycon.com"
```
Comment (guedou) [inline src/views/Home.vue]: Is the website live?

Comment (polybassa) [reply src/views/Home.vue]: This is just the redirect... I need to remove the "s"

Comment (polybassa) [reply src/views/Home.vue]: I fixed it

**Pull request merged**